requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests==2.18.4
 google-cloud-storage==1.8.0
 tenacity==4.10.0
-git+git://github.com/broadinstitute/cromwell-tools.git@v0.3.1
+git+git://github.com/broadinstitute/cromwell-tools.git@v0.5.0
+git+git://github.com/HumanCellAtlas/metadata-api@master#egg=hca-metadata-api[dss]
```

Comment:
We should pin to a specific commit or tag here in case breaking changes are merged into the master branch.